closure/goog/net/streams/jsonstreamparser.js

```diff
@@ -239,7 +239,7 @@ Parser.prototype.getExtraInput = function() {
 
 
 /**
- * @param {string|!ArrayBuffer|!Array<number>} input
+ * @param {string|!ArrayBuffer|!Uint8Array|!Array<number>} input
```

Comment:
This seems unnecessary - it should be fine as simply {string}, since parse() immediately asserts that the input is a string - what directed you to make this change?  If you're seeing a type checking error then it sounds like there's a bug somewhere else.

Reply:
Agreed. Reason is, `error_` is invoked from the `parse` method, which now obviously accepts UInt8Array.
You're right in saying that the first things `parse` does is it asserts the type to a string, but that doesn't change it's type signature, I think a cast would be required and then the signature of `error_` could be stripped down to just string.

...is this flow you guys are using here?

Reply:
No, we use [Closure Compiler](https://github.com/google/closure-compiler).  It recognizes goog.asserts.assertString as narrowing the type of its argument to be only string, which is why I was confused about where you were actually seeing the error.  What type checker are you using?

Reply:
@shicks it was the Travis build that failed. Can't find the failed build in the history though.